### PR TITLE
Javascript documentation

### PR DESCRIPTION
**Pull request by lmlynik**

### CODE REVIEW

File: javaScript.md [hunk 1]
```diff
@@ -3,6 +3,288 @@ title: JavaScript
...
+Or in a html document
+
+```html
+<script src="../../dist/web/matchmore.js"></script>
+```
+
+
+#### Get the API key
+
+Setup application API key and world, get it for free from [http://matchmore.io/](http://matchmore.io/).
```
Comment (elsener) [inline javaScript.md]: should be https?

File: javaScript.md [hunk 2]
```diff
@@ -3,6 +3,288 @@ title: JavaScript
...
+                    altitude: altitude || 0,
+                  }
+                    let publication = manager.createPinDevice(
+                        "Our test pin",
+                        coords,
+                    ).then(pin => {
+                        let p1 = manager.createPublication(
+                            "my-topic",
+                            100 /* m */,
+                            20 /* s */,
```
Comment (wenjdu) [inline javaScript.md:145]: I like how you added the / m and s . I will keep this idea for the other sdks comment.

File: javaScript.md [hunk 3]
```diff
@@ -3,6 +3,288 @@ title: JavaScript
...
+       { "age": 17, "name": "Alex" },
+       pin.id);
+   return;
+}
+
+```
+
+### Main device
+
+First you need to create the main device, first device you create is the main device
```
Comment (wenjdu) [inline javaScript.md]: The sentence is repeated.

File: javaScript.md [hunk 4]
```diff
@@ -3,6 +3,288 @@ title: JavaScript
...
+
+```javascript
+//use your own application specific names
+await manager.createMobileDevice("me", "iOS", "");
```
Comment (wenjdu) [inline javaScript.md]: add the hint for the third parameter - deviceToken

File: javaScript.md [hunk 5]
```diff
@@ -3,6 +3,288 @@ title: JavaScript
...
+await manager.updateLocation({
+    latitude = 54.414662,
+    longitude = 18.625498
+}, deviceId);
+```
+
+#### Create a Mobile Device
+
+```javascript
+let newDevice = await manager.createMobileDevice("another mobile", "android");
```
Comment (wenjdu) [inline javaScript.md:234]: third parameter about fcm and apns ?

**Review by wenjdu — CHANGES_REQUESTED** [review]
It is a great one, 
good job Lukasz !!

**Review by wenjdu — APPROVED** [review]

**Comment by wenjdu** [comment]
Nicely done @lmlynik ! I am merging this.

**Pull request merged**